src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerBinderDelegateProvider.cs

```diff
@@ -100,7 +100,7 @@ public static class ControllerBinderDelegateProvider
             }
         }
 
-        private static BindingInfo[] GetParameterBindingInfo(
+        private static ControllerParameterBindingInfo[] GetParameterBindingInfo(
```

Comment:
What's the reason for this? Just clarity since bindinginfo is already a thing elsewhere?

Reply:
Yeah. I was super confused as to why we were rebuilding `BindingInfo` here when I glanced at the code.